### PR TITLE
Targeting UX - first pass

### PR DESCRIPTION
**Pull request by RichieAHB**
This removes the editable title for targeting (which appears to not be used and is confusing) and also add some UI descriptions around what targets are and how they work. Specifically with an aim to be more clear about where the tags are `AND` and where they are `OR` for showing in proposer. Thoughts?!

targeting

### CODE REVIEW

File: public/video-ui/src/components/Targeting/Targeting.js [hunk 1]
```diff
@@ -44,25 +43,34 @@ class Targeting extends React.Component {
...
+                    data={target}
+                    updateData={this.updateTarget}
+                    editable={true}
+                    formName="TargetingForm"
                   >
-                    <TextInput />
-                  </ManagedField>
-                  {!isDeleting(target, this.props.deleting) && (
+                    <p>
+                      {index === 0 ? (
```
Comment (akash1810) [inline public/video-ui/src/components/Targeting/Targeting.js]: lovely...

**Comment by paperboyo** [comment]
👏 
ActiveUntil > Active Until

------------TO IGNORE-------------

Personally, I’m not sure about the wording, but I am Polish, mind you. It may suggest the article is doing the suggesting and not that it’s the one being suggested to.

”Targeting tags” > ”Target**ed** tags” (or ”Tags to target”)
”Search commissioning info (type ‘\*’ to show all)" > "Target these tags (type ’\*’ for wildcard)”?
”atom” > ”video”

There may be some confusion arising from the fact that what you _target_ end up being _suggested_ by a thing called _proposer_, but that’s not really related to this PR. And dropping one (two?!) of these terms... would be serious business.

Plus wording’s bit wordy?

**Comment by RichieAHB** [comment]
@paperboyo completely agree with all of that. I've made some edits to reflect some of your points. I think some of the larger things around what we mean by targeted / targeting will feed into a wider chat around targeting. As I think there are quite a few misconceptions about it!

Also it is a bit wordy but I can't think of a way of being less wordy and I think it probably is clearer than no words at all??
